jpasskit/src/main/java/de/brendamour/jpasskit/apns/PKSendPushNotificationUtil.java

```diff
@@ -33,45 +34,55 @@
 import com.turo.pushy.apns.util.concurrent.PushNotificationFuture;
 import com.turo.pushy.apns.util.concurrent.PushNotificationResponseListener;
 
+import org.apache.commons.lang3.tuple.Pair;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static de.brendamour.jpasskit.util.Assert.state;
+import static de.brendamour.jpasskit.util.CertUtils.extractApnsTopics;
+import static de.brendamour.jpasskit.util.CertUtils.extractCertificateWithKey;
+import static de.brendamour.jpasskit.util.CertUtils.toInputStream;
+import static de.brendamour.jpasskit.util.CertUtils.toKeyStore;
+
 public class PKSendPushNotificationUtil implements AutoCloseable {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(PKSendPushNotificationUtil.class);
     private static final String EMPTY_PUSH_JSON_STRING = "{}";
     private static final int POOL_SIZE_DEFAULT = 10;
 
     private ApnsClient client;
+    private Set<String> topics;
 
-    public PKSendPushNotificationUtil(final String pathToP12, final String passwordForP12) throws IOException {
-        this(pathToP12, passwordForP12, POOL_SIZE_DEFAULT);
+    public PKSendPushNotificationUtil(String keyStorePath, char [] keyStorePassword) throws IOException {
```

Comment:
Any reason why we can't keep using the pkcs12 file? That's what you export from the macOS Keychain so it would be easiest for everyone to use.
Either way we would need to deprecate the API and offer both options for a while

Reply:
I just renamed the variables to keep them self-descriptive. I call it self-descriptive because we use them to init `KeyStore` entity in Java (according to the existing functionality, not me invention).

Also I see p12 as just a specific format of a more general "keystore" concept, so that's why I prefer this name.

So to sum up:
1. No functional changes or API changes, just renaming - we still work with p12 files. I even have tests to prove this.
2. I may be wrong, so if you insist that it's not a good idea, I'll revert this renaming back.

Reply:
ok. that makes sense